PYCS_Language/Operations/BooleanOperationHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿namespace SplitAndMerge
+{
+    class BooleanOperationHandler
+    {
+        public bool CompareValues(string str, Dictionary<string, double> vars)
+        {
+            str = str.Trim();
```

Comment:
I suggest u start using StringBuilder instead of string in some cases. Cuz string is immutable (every change in string will create new string and that's not pretty good from memory economy prospective)

https://www.shekhali.com/difference-between-string-and-stringbuilder-in-csharp/